include/grpc++/server.h

```diff
@@ -136,12 +136,17 @@ class Server final : public ServerInterface, private GrpcLibraryCodegen {
   /// completion queue (in param sync_server_cqs) to use for listening to
   /// incoming requests (used only in case of sync server)
   ///
+  /// \param max_threads The maximum number of threads per server completion
+  /// queue (i.e number of polling threads and the number of threads executing
+  /// the RPC handlers)
+  ///
```

Comment:
This is an API extension and has to be something that we are permanently ok with. As such, can I suggest going through the gRFC process here? I know that this is a long-standing feature request, but there are possibly other ways of doing this and also providing other benefits without going through this particular API extension. For example, there are other variants of the feature request (#12472, #12737) that maybe could get handled by a unified path or at the core level instead of the C++ level. Additionally, the PR in progress (#13697) will have a side effect of generating a recoverable RESOURCE_EXHAUSTED status instead of exception when we are out of threads.